cmd/kube-apiserver/app/options/options.go

```diff
@@ -231,4 +234,10 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.EnableAggregatorRouting, "enable-aggregator-routing", s.EnableAggregatorRouting,
 		"Turns on aggregator routing requests to endoints IP rather than cluster IP.")
 
+	fs.StringVar(&s.IDTokenIssuerKey, "id-token-issuer-key", s.IDTokenIssuerKey, ""+
```

Comment:
what are the planned flags for the public keyset used for verification, and the list of accepted issuers? it would be good to explicitly connect this (and the issuer id) to the tokenrequest API

Reply:
Thinking about this more, the flags I want to add are:
```
--service-account-signing-key-file
--service-account-issuer
--service-account-api-audiences
```
`issuer` and `api-audience` fit nicely in `BuiltInAuthenticationOptions`. I'm not sure where `signing-key-file` should go.

I want to reuse:
```
--service-account-lookup
--service-account-key-file
```
to configure the new authenticator. We can consider changing the name of `key-file` to `public-key-file` somewhere down the line.